### PR TITLE
Update Effect: Tenacious Flames and Effect: Impaling Chain to use "to-failure" rather than "two-degrees-worse"

### PR DESCRIPTION
**Pull request by simonward**

### CODE REVIEW

File: static/lang/en.json [hunk 1]
```diff
@@ -3928,6 +3932,8 @@
         "TraitDescriptionWitch": "This indicates abilities from the witch class.",
         "TraitDescriptionWizard": "This indicates abilities from the wizard class.",
         "TraitDescriptionWood": "Effects with the wood trait conjure or manipulate wood. Those that manipulate wood have no effect in an area without wood. Creatures with this trait consist primarily of wood or have a connection to magical wood.",
+        "TraitBottledBreath": "Bottled Breath",
```
Comment (TikaelSol) [inline static/lang/en.json:3935]: Alphabetize

File: src/scripts/config/traits.ts [hunk 2]
```diff
@@ -457,6 +457,7 @@ const weaponTraits = {
     plant: "PF2E.TraitPlant",
     poison: "PF2E.TraitPoison",
     propulsive: "PF2E.TraitPropulsive",
+    radiation: "PF2E.TraitRadiation",
```
Comment (TikaelSol) [inline src/scripts/config/traits.ts:460]: Not needed in weapon traits, it should go in npc attack traits.

File: static/lang/en.json (hunk 1 above)
Comment (simonward) [reply static/lang/en.json:3935]: It's sorted with like. Should I sort the whole list?

Comment (TikaelSol) [reply static/lang/en.json:3935]: The whole list should be alphabetized, I did a sort on it not long ago but for the love of everything holy don't sort the whole thing right now 😄 My RoE branch has to be able to merge without me throwing a monitor out a window on the fifteenth merge conflict.

**Pull request merged**